README.md

```diff
@@ -0,0 +1,6 @@
+The application takes in a file path as input
+
+It is assumed that the user specifies a directory filled with movies
```

Comment:
Nice README file. To format your READMEs better, there is a syntax called Markdown syntax. Check it out - you can make your README look prettier.